tests/new_core/processors/test_warming_level.py

```diff
@@ -343,7 +343,7 @@ def test_empty_warming_level_times_error(self, mock_read_csv_file, processor):
     def test_execute_updates_context(self, request, full_processor):
         """Test that execute updates context with warming level information."""
         test_result = request.getfixturevalue("test_dataarray_dict")
-        context = {}
+        context = {"activity_id": "WRF"}
```

Comment:
This is missing a LOCA test, I will need to think about how to make this additional LOCA test cleanly.

Reply:
Also needs an accuracy test now, since we want to make sure start and end year cutoffs are returning expected WL slices.